### ISSUE TITLE
Panick when using relative pathname

### ISSUE THREAD

**Issue by ngirard** [dalance/amber#36]
Using Anchor v0.5.5 (precompiled binary) on Ubuntu 20.04.

Doing
```bash
echo Test > f
ambs -r -- 'Test' f
```
results in
```
thread 'finder' panicked at 'called `Result::unwrap()` on an `Err` value: Os { code: 2, kind: NotFound, message: "No such file or directory" }', src/pipeline_finder.rs:133:42
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

The behaviour is the same with `ambr`.

Invoking both programs with `./f` instead of `f` works fine.

**Comment by dalance** [comment]
Thanks! I fixed it, and released v0.5.7.

**Comment by ngirard** [comment]
Great, thanks !
